src/Card/Card.stories.js

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import { Card } from "./Card";
+import { cardAlignOptions } from "./Card.levers";
+import { Heading } from "../Heading/Heading";
+import { CtaLink } from "../CtaLink/CtaLink";
+import { CtaButton } from "../CtaButton/CtaButton";
+
+export default {
+  title: "Composite/Card",
+  component: Card,
+  subcoponent: { Heading, CtaLink, CtaButton },
+  argTypes: {
+    align: {
+      control: {
+        type: "inline-radio",
+        options: cardAlignOptions,
+      },
+    },
+    isMinimal: {
+      control: {
+        type: "boolean",
+      },
+    },
+  },
+};
+
+const CardTemplate = ({ ...rest }) => (
+  <Card {...rest}>
+    {/* Developer would pass in their own card content components */}
+    <span className="su-font-sans su-font-bold su-type-0 su-mb-0">
+      Super headline
+    </span>
+    <Heading level="3" size={1} className="su-mb-0">
+      Headline
+    </Heading>
+    <p className="su-card-paragraph">
+      Fusce convallis metus id felis luctus adipiscing. Vestibulum suscipit
+      nulla quis orci. Phasellus a est.
+    </p>
+    <CtaLink
+      text="Learn more"
+      srText="about Decanter"
+      animate="right"
+      icon="more"
+      href="https://decanter.stanford.edu"
+    />
+    <CtaButton
+      text="Learn more"
+      srText="about Decanter"
+      animate="right"
+      icon="more"
+      variant="solid"
+      size="default"
+      href="https://decanter.stanford.edu"
+      className="su-mt-16"
+    />
```

Comment:
Would we want to wrap this in a containing HTML element so it is easier to move around later with css or no?

Reply:
I see a previous comment from Yvonne that agrees.

Reply:
I could see it being helpful to wrap it, so for example, if it's a minimal card, then no padding on this wrapper. Totally open to ideas.

Reply:
Haha we posted at the same time @sherakama